dotcom-rendering/src/components/FlexibleGeneral.tsx

```diff
@@ -269,6 +263,12 @@ export const SplashCardLayout = ({
 		);
 	}
 
+	const useLargerHeadlineSizeDesktop =
+		// When there's no image, we want the text to take up more space. The exception is Opinion
+		// cards, as avatars are more common and command less visual weight that a standard image.
+		(!card.image && card.format.design !== ArticleDesign.Comment) ||
```

Comment:
Don't we want to keep the imageSuppressed check for non-Opinion pieces?

Reply:
Yeah I've just extracted out `imageSuppressed` from `decideSplashCardProperties()` into this variable. If there is no image and it's not an opinion piece, the larger headline sizes will be used at the desktop breakpoint.

Reply:
Got it. I had thought `imageSuppressed` was some kind of flag in addition to `card.image`. But I can see it's derived from `!card.image` being passed through so the two are equivalent 👍